Source/WTF/config.h

```diff
@@ -29,6 +29,7 @@
 #undef new
 #undef delete
 #include <wtf/FastMalloc.h>
+#include <wtf/TZoneMalloc.h>
```

Comment:
It would be nice to have data to tell us if this actually helps.

Reply:
This one is not currently needed.  I included it for consistency so that anyone adding uses of TZoneHeap macros  in WTF code won't be surprised why their code doesn't build.  If you have data showing that this severely impacts build speed, I can omit it for the WTF case for now.